src/components/vistas/tablet/container-card/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import Card from './card';
+
+class ContainerCard extends React.Component {
+    render() {
+        return(
+            <div className={styles.ContainerCard}>
```

Comment:
Mantener el casing consistente.
```suggestion
            <div className={styles.containerCard}>
```

Reply:
listo